pkg/provision/storage/shared.go

```diff
@@ -58,18 +58,23 @@ func getCommonPVCSpec(namespace string, size string) (*corev1.PersistentVolumeCl
 	}, nil
 }
 
+// isEphemeral evaluates is volume component is configured as ephemeral
+func isEphemeral(volume dw.VolumeComponent) bool {
```

Comment:
It makes sense for this to take a reference (most of our functions work this way)